README.md

```diff
@@ -63,6 +64,8 @@ Editing assistance for configuration and application files is provided through t
 
 Projects with the Liberty Maven Plugin or Liberty Gradle Plugin configured will appear in the Liberty tool window on the sidebar. If the tool window is not enabled by default, you can view it by selecting **View > Tool Windows > Liberty**.
 
+> Note: Make sure the **JAVA_HOME** system environment variable is set to define the JDK for running Liberty dev mode.
```

Comment:
@dmuelle Please review this latest change reflecting our Slack discussion.

Removed the Settings section in this README as it's covered in user-guide, but mentioned this note here as it is important.

Reply:
do we need to mention Java 17 or later here? Also the previous statement said

> The **JAVA_HOME** system environment variable in the current terminal is used when you run Liberty dev mode in any project

but the user guide doesn't mention Java home

Reply:
Here is fine for the user to use a <17 version of Java, as it is used for running Liberty dev mode.

The plugin runs with the IntelliJ-bundled Java.